### ISSUE TITLE
Why did you ignore the InstanceNorm in the first block of Unet?

### ISSUE THREAD

**Issue by John1231983** [junyanz/pytorch-CycleGAN-and-pix2pix#782]
Could you tell me any reason why do you ignore the instance norm in the fist down-sampling block in the unet? I have added the norm in the block but the performance worst 

https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/43521b0e670f5421487d8aaf3675786e177c72e6/models/networks.py#L514

**Comment by junyanz** [comment]
If you use an instancenorm in the first layer, the color of the input image will be normalized and get ignored. For many applications, you may want to preserve the color of the input image.

**Comment by John1231983** [comment]
Great to know that. How about batch norm? Why I can still use batch norm in the first layer and the result does not reduce. 

Secondly, if I add 3x3 convolution (without instance norm) before the first layer, so Can I use instance norm in the first layer?

**Comment by junyanz** [comment]
(1) The batchnorm calculates the statistics over the entire dataset. Therefore the color of each image is still preserved. (2) You can try it. We haven't tried it before. (3) I am not sure if it is related to normalization. Please see the [discussion](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/issues/190) on checkboard artifacts for more details.
